content/docs/started/getting-started.md

```diff
@@ -11,46 +11,24 @@ This document provides information about setting up Kubeflow in various environm
 It's important that you have some knowledge of the following systems and tools:
 
 * [Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
-* [TensorFlow](https://www.tensorflow.org/get_started/)
-* [kustomize](https://kustomize.io/)
+* [Kustomize](https://kustomize.io/)
 
-### Kubeflow requirements
+## Installing Kubeflow
 
-Before installing Kubeflow, ensure that your environment meets the
-following requirements:
-
-* You need kustomize version {{% kustomize-min-version %}} or later. See the 
-  [kustomize component guide](/docs/components/misc/kustomize/) 
-  for details about installing kustomize.
+There are various ways to install Kubeflow. Choose one of the following options
+to suit your environment ([desktop or server](#node), [Kubernetes cluster or public cloud](#cloud)):
 
-* If you intend to install Kubeflow on an existing Kubernetes cluster, the
```

Comment:
I see why you've removed this info (it's primarily for existing clusters) but I still think it's useful to have the info up front. Users ask frequently which version of K8s they need, for example. Perhaps keep the info here but change the intro wording slightly?

Reply:
I was hoping to keep the information in a single place - ie on the existing Kubernetes page. What do you think about providing a link here to general system requirements? Let me try that.

Reply:
I've added a link.